module/Application/view/partial/main-nav.phtml

```diff
@@ -173,6 +173,7 @@ $lang = $this->plugin('translate')->getTranslator()->getLocale();
                             <li><a href="<?= $this->url('activity_calendar') ?>"> <?= $this->translate('Option
                             calendar') ?></a></li>
                         <?php endif; ?>
+                        <li><a href="<?= $this->url('activity/career') ?>"> <?= $this->translate('Career related') ?></a></li>
```

Comment:
Levi mentioned that a nav-bar link would not be sufficient. He requested that there is a link on the regular activity page, i.e. in the 'body'. Next to the activity creation buttons could work, probably in a different color. Not exactly sure what's best here.